meta-mender-tegra/meta-mender-tegra-jetpack5/classes/tegra-mender-setup.bbclass

```diff
@@ -46,18 +49,19 @@ IMAGE_BOOT_FILES = "u-boot-dtb.bin"
 # boot into an emergency shell and examine the /dev/mmcblk* devices,
 # or use the uboot console to look at mtdparts
 MENDER_DATA_PART_NUMBER_DEFAULT:tegra186 = "34"
-MENDER_DATA_PART_NUMBER_DEFAULT:tegra194 = "43"
+MENDER_DATA_PART_NUMBER_DEFAULT:tegra194 = "42"
```

Comment:
This is where I needed to set `MENDER_DATA_PART_NUMBER` to 45 for a clean boot into our Syslogic BRLA3 Xavier box.

Reply:
This is still to be validated with the Xavier dev kit.

Reply:
@cakre Is this what you are also experiencing with the partition number?

Reply:
Nope, there is another override for `xavier-nx` specifically that sets the data partition to 12 which is quite a bit off. I'll open a draft MR (still figuring out why `initrd-flash` is failing)

Reply:
I have this set to 42 to boot cleanly into our custom Xavier devices.

Reply:
@mwest90 agx-xaver or xavier-nx? I am using a xavier-nx

Reply:
@cakre agx xavier